undefined: errors.ValidatorReason

<!--
Please answer these questions before submitting your issue. Thanks!
For questions please use one of our forums: https://go-kratos.dev/docs/getting-started/faq
-->
#### What happened:
我使用了 `github.com/go-kratos/kratos/contrib/middleware/validate/v2` 库，但是 kratos v2.8.4 errors包还没有`errors.ValidatorReason`这个全局变量。
我查看了只有main分支才有这个变量

```
 could not import github.com/go-kratos/kratos/contrib/middleware/validate/v2 (-: # github.com/go-kratos/kratos/contrib/middleware/validate/v2
../../../go/pkg/mod/github.com/go-kratos/kratos/contrib/middleware/validate/v2@v2.0.0-20250423115721-dbd7664eff95/validate.go:23:43: undefined: errors.ValidatorReason
../../../go/pkg/mod/github.com/go-kratos/kratos/contrib/middleware/validate/v2@v2.0.0-20250423115721-dbd7664eff95/validate.go:30:43: undefined: errors.ValidatorReason) (typecheck)
```
#### What you expected to happen:
修复这个bug
#### How to reproduce it (as minimally and precisely as possible):
使用 `github.com/go-kratos/kratos/contrib/middleware/validate/v2`库
然后使用`validate.ProtoValidate()`中间件
#### Anything else we need to know?:
有可能是因为 validate/v2库昨天才发布了新版本，kratos版本还没来得及发布新版本。或者说我应该用旧版本的validate?
#### Environment:
- Kratos version (use `kratos -v`):  v2.8.4
- Go version (use `go version`):  go1.24.2 darwin/arm64
- OS (e.g: `cat /etc/os-release`): macos 15.4.1
- Others:

`github.com/go-kratos/kratos/contrib/middleware/validate/v2 v2.0.0-20250423115721-dbd7664eff95`
`github.com/go-kratos/kratos/v2 v2.8.4`

--- a/contrib/middleware/validate/validate.go
+++ b/contrib/middleware/validate/validate.go
@@ -20,14 +20,14 @@ func ProtoValidate() middleware.Middleware {
 		return func(ctx context.Context, req any) (reply any, err error) {
 			if msg, ok := req.(proto.Message); ok {
 				if err := protovalidate.Validate(msg); err != nil {
-					return nil, errors.BadRequest(errors.ValidatorReason, err.Error()).WithCause(err)
+					return nil, errors.BadRequest("VALIDATOR", err.Error()).WithCause(err)
 				}
 			}
 
 			// to compatible with the [old validator](https://github.com/envoyproxy/protoc-gen-validate)
 			if v, ok := req.(validator); ok {
 				if err := v.Validate(); err != nil {
-					return nil, errors.BadRequest(errors.ValidatorReason, err.Error()).WithCause(err)
+					return nil, errors.BadRequest("VALIDATOR", err.Error()).WithCause(err)
 				}
 			}
 

--- a/contrib/polaris/ratelimit.go
+++ b/contrib/polaris/ratelimit.go
@@ -16,7 +16,7 @@ import (
 
 // ErrLimitExceed is service unavailable due to rate limit exceeded.
 var (
-	ErrLimitExceed = errors.New(429, errors.RateLimitReason, "service unavailable due to rate limit exceeded")
+	ErrLimitExceed = errors.New(429, "RATELIMIT", "service unavailable due to rate limit exceeded")
 )
 
 // Ratelimit Request rate limit middleware

--- a/errors/errors.go
+++ b/errors/errors.go
@@ -15,12 +15,6 @@ const (
 	UnknownCode = 500
 	// UnknownReason is unknown reason for error info.
 	UnknownReason = ""
-	// ValidatorReason indicates an error related to request validation.
-	ValidatorReason = "VALIDATOR"
-	// CodecReason indicates an error related to encoding/decoding.
-	CodecReason = "CODEC"
-	// RateLimitReason indicates an error caused by rate limiting.
-	RateLimitReason = "RATELIMIT"
 	// SupportPackageIsVersion1 this constant should not be referenced by any other code.
 	SupportPackageIsVersion1 = true
 )

--- a/middleware/ratelimit/ratelimit.go
+++ b/middleware/ratelimit/ratelimit.go
@@ -11,7 +11,7 @@ import (
 )
 
 // ErrLimitExceed is service unavailable due to rate limit exceeded.
-var ErrLimitExceed = errors.New(429, errors.RateLimitReason, "service unavailable due to rate limit exceeded")
+var ErrLimitExceed = errors.New(429, "RATELIMIT", "service unavailable due to rate limit exceeded")
 
 // Option is ratelimit option.
 type Option func(*options)

--- a/middleware/validate/validate.go
+++ b/middleware/validate/validate.go
@@ -19,7 +19,7 @@ func Validator() middleware.Middleware {
 		return func(ctx context.Context, req any) (reply any, err error) {
 			if v, ok := req.(validator); ok {
 				if err := v.Validate(); err != nil {
-					return nil, errors.BadRequest(errors.ValidatorReason, err.Error()).WithCause(err)
+					return nil, errors.BadRequest("VALIDATOR", err.Error()).WithCause(err)
 				}
 			}
 			return handler(ctx, req)

--- a/transport/http/binding/bind.go
+++ b/transport/http/binding/bind.go
@@ -12,7 +12,7 @@ import (
 // BindQuery bind vars parameters to target.
 func BindQuery(vars url.Values, target any) error {
 	if err := encoding.GetCodec(form.Name).Unmarshal([]byte(vars.Encode()), target); err != nil {
-		return errors.BadRequest(errors.CodecReason, err.Error())
+		return errors.BadRequest("CODEC", err.Error())
 	}
 	return nil
 }
@@ -23,7 +23,7 @@ func BindForm(req *http.Request, target any) error {
 		return err
 	}
 	if err := encoding.GetCodec(form.Name).Unmarshal([]byte(req.Form.Encode()), target); err != nil {
-		return errors.BadRequest(errors.CodecReason, err.Error())
+		return errors.BadRequest("CODEC", err.Error())
 	}
 	return nil
 }

--- a/transport/http/binding/bind_test.go
+++ b/transport/http/binding/bind_test.go
@@ -49,7 +49,7 @@ func TestBindQuery(t *testing.T) {
 				vars:   map[string][]string{"age": {"kratos"}, "url": {"https://go-kratos.dev/"}},
 				target: &TestBind2{},
 			},
-			err: kratoserror.BadRequest(kratoserror.CodecReason, "Field Namespace:age ERROR:Invalid Integer Value 'kratos' Type 'int' Namespace 'age'"),
+			err: kratoserror.BadRequest("CODEC", "Field Namespace:age ERROR:Invalid Integer Value 'kratos' Type 'int' Namespace 'age'"),
 		},
 		{
 			name: "test2",
@@ -118,7 +118,7 @@ func TestBindForm(t *testing.T) {
 				},
 				target: &TestBind2{},
 			},
-			err:  kratoserror.BadRequest(kratoserror.CodecReason, "Field Namespace:age ERROR:Invalid Integer Value 'a' Type 'int' Namespace 'age'"),
+			err:  kratoserror.BadRequest("CODEC", "Field Namespace:age ERROR:Invalid Integer Value 'a' Type 'int' Namespace 'age'"),
 			want: nil,
 		},
 	}

--- a/transport/http/codec.go
+++ b/transport/http/codec.go
@@ -61,7 +61,7 @@ func DefaultRequestQuery(r *http.Request, v any) error {
 func DefaultRequestDecoder(r *http.Request, v any) error {
 	codec, ok := CodecForRequest(r, "Content-Type")
 	if !ok {
-		return errors.BadRequest(errors.CodecReason, fmt.Sprintf("unregister Content-Type: %s", r.Header.Get("Content-Type")))
+		return errors.BadRequest("CODEC", fmt.Sprintf("unregister Content-Type: %s", r.Header.Get("Content-Type")))
 	}
 	data, err := io.ReadAll(r.Body)
 
@@ -69,13 +69,13 @@ func DefaultRequestDecoder(r *http.Request, v any) error {
 	r.Body = io.NopCloser(bytes.NewBuffer(data))
 
 	if err != nil {
-		return errors.BadRequest(errors.CodecReason, err.Error())
+		return errors.BadRequest("CODEC", err.Error())
 	}
 	if len(data) == 0 {
 		return nil
 	}
 	if err = codec.Unmarshal(data, v); err != nil {
-		return errors.BadRequest(errors.CodecReason, fmt.Sprintf("body unmarshal %s", err.Error()))
+		return errors.BadRequest("CODEC", fmt.Sprintf("body unmarshal %s", err.Error()))
 	}
 	return nil
 }

